components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -368,9 +470,7 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
         let (index, term) = self.apply_progress();
         apply_res.applied_index = index;
         apply_res.applied_term = term;
-        if self.reset_state_changed() {
```

Comment:
Question: Why is it removed?

Reply:
Because the state is updated in place, no need to detect changes.